functional_tests/grpc/01-simple.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+import os
+
+import wandb
+
+# Temporary environment variable for testing grpc service mode
+os.environ["WANDB_SERVICE_GRPC"] = "true"
+
+wandb.require("service")
```

Comment:
should we just set the env variable when we call `wandb.require("service")`?

Reply:
This is the special case, where we are choosing grpc mode just to keep that code path tested.

The normal case will not use gRPC in "wandb service"

we might want to make it:
`wandb.require("grpc")` instead.... ill think about it tonight when i finish this part of the code and the tests.

It would be nice if we could run all the functional_tests/mp tests with both modes, but yea doesnt quite support this right now

Reply:
the quick and dirty way to test this (temporarily until we can make yea tests more composable - [ticket](https://github.com/wandb/yea-wandb/issues/11)):

Just make grpc a copy of the mp directory with the shard changed and some environment variable set.   Maybe ill put a script in the grpc directory so at least it is automated.